osu.Game/Screens/Edit/Screens/Setup/Components/LinkSpriteText.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Input.States;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using System.Diagnostics;
+
+namespace osu.Game.Screens.Edit.Screens.Setup.Components
+{
+    public class LinkSpriteText : OsuHoverContainer
+    {
+        private readonly OsuSpriteText label;
+
+        private const float default_text_size = 12;
```

Comment:
not using `int` here?

Reply:
Since text sizes and sizes in general are using `float`s, for consistency I am using `float`s.